From: Markus Koschany <apo@debian.org>
Date: Mon, 15 Feb 2016 00:37:42 +0100
Subject: exclude unsupported classes from antlib.xml

Exclude unsupported classes from antlib.xml. Enabling them causes build
failures in projects that depend on ivyplusplus.

Forwarded: not-needed
---
 src/com/zwitserloot/ivyplusplus/antlib.xml | 5 -----
 1 file changed, 5 deletions(-)

--- a/src/com/zwitserloot/ivyplusplus/antlib.xml
+++ b/src/com/zwitserloot/ivyplusplus/antlib.xml
@@ -22,18 +22,11 @@
 -->
 <antlib xmlns:current="ant:current">
 	<!-- insert here -->
-	<typedef name="eclipsegen" classname="com.zwitserloot.ivyplusplus.eclipse.BuildEclipseProject" />
-	<typedef name="intellijgen" classname="com.zwitserloot.ivyplusplus.intellij.BuildIntellijProject" />
-	<typedef name="eclipse-gen" classname="com.zwitserloot.ivyplusplus.eclipse.BuildEclipseProject" />
-	<typedef name="intellij-gen" classname="com.zwitserloot.ivyplusplus.intellij.BuildIntellijProject" />
 	<typedef name="compile" classname="com.zwitserloot.ivyplusplus.Compile" />
 	<typedef name="ensureippversion" classname="com.zwitserloot.ivyplusplus.EnsureVersion" />
 	<typedef name="ensure-ipp-version" classname="com.zwitserloot.ivyplusplus.EnsureVersion" />
-	<typedef name="create-artifact-bundle" classname="com.zwitserloot.ivyplusplus.mavencentral.CreateArtifactBundle" />
 	<typedef name="cachedunjar" classname="com.zwitserloot.ivyplusplus.CachedUnjar" />
 	<typedef name="hardlink" classname="com.zwitserloot.ivyplusplus.Hardlink" />
-	<typedef name="scpUpload" classname="com.zwitserloot.ivyplusplus.ssh.ScpUpload" />
-	<typedef name="sshExec" classname="com.zwitserloot.ivyplusplus.ssh.SshExec" />
 	
 	<macrodef name="show-html" description="Shows an HTML page.">
 		<attribute name="file" />
